docs/plugins.md

```diff
@@ -240,29 +309,134 @@ Sample:
 }
 ```
 
-* With REST protocol, we use the routes configured in _routes.js_, prefixed by "\_plugin/" + the plugin name:<br>
+* With REST protocol, we use the routes configured in _routes.js_.  
+These routes are automatically prefixed with "\_plugin/" + the plugin name.
+
 Samples:
 
 GET action:
 
 ```
-GET http://kuzzle:7512/api/_plugin/myplugin/foo/John%20Doe
+GET http://kuzzle:7511/api/_plugin/myplugin/foo/John%20Doe
```

Comment:
should fix URL with api version :  http://kuzzle:7511/api/1.0/_plugin/myplugin/foo/John%20Doe